core/src/test/java/org/apache/calcite/sql/dilect/HiveSqlDialectTest.java

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
You can test the function in RelToSqlConverterTest. A new HiveSqlDialectTest is not necessary.

Reply:
Thanks for advise. 
I have put test in RelToSqlConverterTest.